tritium-metrics/src/test/java/com/palantir/tritium/metrics/MetricRegistriesTest.java

```diff
@@ -394,6 +396,42 @@ public void problematicTimestamps() {
         assertParsesTimestamp("2019-03-29T23:38:51.092Z");
     }
 
+    @Test
+    public void codahale_registry_histogram_count_should_monotonically_increase_after_window()
+            throws InterruptedException {
+        MetricRegistry registry = new MetricRegistryWithReservoirs(
+                () -> new SlidingTimeWindowArrayReservoir(1, TimeUnit.MILLISECONDS));
```

Comment:
can we verify this with `HdrHistogram` reservoir as well?

Reply:
Tbh I was going to just delete these two tests as I think the ones above did actually capture behaviour, they just captured the undesired behaviour!

Not sure what Thread.sleep we'd want on the HDR one - I thought it doens't rely on a clock?